src/core/public/chrome/ui/header/collapsible_nav_group_enabled_top.tsx

```diff
@@ -148,10 +149,10 @@ export const CollapsibleNavTop = ({
           />
         </EuiFlexItem>
       </EuiFlexGroup>
-      {currentNavGroup?.title && (
+      {currentNavGroup?.title && collapsibleNavHeaderRender && (
```

Comment:
```suggestion
      {collapsibleNavHeaderRender && (
```
The title should be useless here.